com.ibm.wala.cast.java.test.data/build.xml

```diff
@@ -78,8 +68,8 @@
 	   excludes="**/Activator.java"/>
   </target>
   
-  <target name="jar" depends="compile,testdatadir">
-    <jar destfile="${basedir}/../com.ibm.wala.ide.jdt.test/testdata/test_project.zip" manifest="${basedir}/META-INF/MANIFEST.MF" basedir="${basedir}"/>
+  <target name="jar" depends="compile">
```

Comment:
I suspect this was the main reason we had to run `mvn verify` twice to get the tests to pass before.  This script was writing a zip file directly to `com.ibm.wala.ide.jdt.test` but that project declared no dependence on this one.  I've fix things so the zip file gets written locally and then copied over by `com.ibm.wala.ide.jdt.test`, and that a dependence exists in the build.